### PR TITLE
translate: add clean-cache.chinese.md

### PR DESCRIPTION
**Pull request by MattJin**

### CODE REVIEW

File: sections/docker/clean-cache.chinese.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+*请注意如果您使用multi-stage构建，只要您在最后阶段不安装新的包，清除缓存是没意义的*
+
+<br/><br/>
+
+### 代码示例 - 清除缓存
+
+<details>
+<summary><strong>Dockerfile</strong></summary>
+
+```
```
Comment (rluvaton) [inline sections/docker/clean-cache.chinese.md:18]: Could you please add `dockerfile` language here 😀

**Review by rluvaton — CHANGES_REQUESTED** [review]
Thank you so much!, I'll ask someone to review it 😀

**Comment by rluvaton** [comment]
@goldbergyoni who do we have that can review it?

**Comment by goldbergyoni** [comment]
@rluvaton Matt is the author of the origin Chinese translation, he has 20% shares of this repo 😅

@MattJin

**Review by goldbergyoni — DISMISSED** [review]

File: sections/docker/clean-cache.chinese.md (hunk 1 above)
Comment (goldbergyoni) [reply sections/docker/clean-cache.chinese.md:18]: @rluvaton @MattJin After reviewing I noticed that, let's make this minor fix and merge. Thanks both! 💪

**Review by MattJin — COMMENTED** [review]
plz have a check

File: sections/docker/clean-cache.chinese.md (hunk 1 above)
Comment (MattJin) [reply sections/docker/clean-cache.chinese.md]: done

**Review by rluvaton — APPROVED** [review]

**Comment by rluvaton** [comment]
Thank you so much

**Pull request merged**